tests/v2/validation/charts/gatekeeperallowednamespacespostupgrade_test.go

```diff
@@ -0,0 +1,34 @@
+package charts
+
+import (
+	"strings"
+
+	settings "github.com/rancher/rancher/pkg/settings"
+	namespaces "github.com/rancher/rancher/tests/framework/extensions/namespaces"
+	"github.com/rancher/rancher/tests/framework/pkg/environmentflag"
+	"github.com/stretchr/testify/require"
+)
+
+func (n *GateKeeperAllowedNamespacesTestSuite) TestGateKeeperAllowedNamespacesPostUpgrade() {
+
+	subSession := n.session.NewSession()
+	defer subSession.Cleanup()
+
+	client, err := n.client.WithSession(subSession)
+	require.NoError(n.T(), err)
+
+	if !client.Flags.GetValue(environmentflag.GatekeeperAllowedNamespaces) {
+		n.T().Skip()
```

Comment:
Same nitpick as above, more logging about skip reason might be better while running the tests

Reply:
added logging